src/Symfony/Component/Translation/Tests/Extractor/PhpExtractorTest.php

```diff
@@ -69,11 +69,15 @@ public function testExtraction($resource)
         $actualCatalogue = $catalogue->all();
 
         $this->assertEquals($expectedCatalogue, $actualCatalogue);
+
+        $filename = __DIR__.'/../fixtures/extractor'.\DIRECTORY_SEPARATOR.'translation.html.php';
```

Comment:
IIUC, the filename might mix both `/` and `\\` (on Windows at least). Can we standardize to always use `/` instead?

Reply:
Yes. Good idea!